### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by lilsweetcaligula**
#64 #65 #66 #67

WARNING: `seneca-store-test` cannot be released until:
- `seneca-mem-store@6.0.0` is released

### CODE REVIEW

File: store-test.js [hunk 1]
```diff
@@ -1509,7 +1555,7 @@ function upserttest(settings) {
                       }
 
                       try {
```
Comment (rjrodger) [inline store-test.js:1557]: these try-catch blocks serve no purpose

File: store-test.js [hunk 2]
```diff
@@ -426,6 +434,44 @@ function basictest(settings) {
...
+        var product = si.make('products')
+        product.id = 'product-to-be-updated'
+        product.label = 'lorem ipsum'
+
+        product.save$(function (err, out) {
+          if (err) {
+            return done(err)
+          }
+
+          try {
```
Comment (rjrodger) [inline store-test.js:447]: remove these try-catch blocks to remain consistent with existing test code

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: store-test.js (hunk 2 above)
Comment (lilsweetcaligula) [reply store-test.js:447]: Will do. However without these blocks the test output in case of failure may be imprecise due to the asynchronous nature.

Comment (lilsweetcaligula) [reply store-test.js]: @rjrodger Done.

File: store-test.js (hunk 1 above)
Comment (lilsweetcaligula) [reply store-test.js]: @rjrodger Done.

**Pull request merged**